apollo-composition/src/lib.rs

```diff
@@ -161,12 +159,13 @@ pub trait HybridComposition {
     /// 9. <connectors> expand supergraph
     /// 10. Validate satisfiability
     ///
+    /// If successful, returns a tuple of (the supergraph SDL, a list of hints).
     /// In case of a composition failure, we return a list of errors from the current composition
     /// phase.
     async fn experimental_compose(
         mut self,
         subgraph_definitions: Vec<SubgraphDefinition>,
-    ) -> Result<PluginResult, Vec<Issue>>
+    ) -> Result<(String, Vec<Issue>), Vec<Issue>>
```

Comment:
IMHO returning proper type is clearer than a tuple, i.e. what is the meaning behind `String` and `Vec<Issue>` that are being returned?